scripts/publish-module.gradle

```diff
@@ -24,6 +24,13 @@ afterEvaluate {
                             url = 'https://github.com/processout/processout-android/blob/master/LICENSE'
                         }
                     }
+                    developers {
+                        developer {
+                            id = 'vitalii-vanziak-cko'
+                            name = 'Vitalii Vanziak'
+                            email = 'vitalii.vanziak@processout.com'
```

Comment:
I suggest to use GitHub secrets and pass this info through env. Ideally it should be service account, but for now moving hardcode from git sounds acceptable

Reply:
done